src/plugins/data_view_field_editor/public/components/field_format_editor/field_format_editor.tsx

```diff
@@ -121,6 +120,8 @@ export class FormatSelectEditor extends PureComponent<
       ? fieldFormats.getInstance(value?.id, value?.params)
       : fieldFormats.getDefaultInstance(kbnType, esTypes);
 
+    const fieldFormatParams = format.params();
```

Comment:
These caused https://github.com/elastic/kibana/issues/121170 in the field format editor.

The difference between raw `params` and `params()` is that `params()` would return default values for a field format type. We need to pass the into an editor to avoid desync of state between the form and underlying field format state